.github/workflows/integration_test.yaml

```diff
@@ -26,7 +26,9 @@ jobs:
       - name: Install dependencies
         run: |
           sudo apt update
+          sudo apt upgrade
```

Comment:
do we need to apply upgrades after all, or was it enough to add libunwind-dev?

Reply:
I don't think we need it after all - I'll remove it and check